components/Dialogs/src/MDCDialogTransitionController.m

```diff
@@ -60,30 +93,46 @@ - (void)animateTransition:(id<UIViewControllerContextTransitioning>)transitionCo
   CGFloat endingAlpha = presenting ? 1 : 0;
 
   animatingView.frame = [transitionContext finalFrameForViewController:animatingViewController];
-  animatingView.alpha = startingAlpha;
 
-  NSTimeInterval transitionDuration = [self transitionDuration:transitionContext];
   UIViewAnimationOptions options =
       UIViewAnimationOptionAllowUserInteraction | UIViewAnimationOptionBeginFromCurrentState;
 
-  [UIView animateWithDuration:transitionDuration
-      delay:0.0
-      options:options
-      animations:^{
-        animatingView.alpha = endingAlpha;
-      }
-      completion:^(__unused BOOL finished) {
-        // If we're dismissing, remove the presented view from the hierarchy
-        if (!presenting) {
-          [fromView removeFromSuperview];
-        }
-
-        // From ADC : UIViewControllerContextTransitioning
-        // When you do create transition animations, always call the
-        // completeTransition: from an appropriate completion block to let UIKit know
-        // when all of your animations have finished.
-        [transitionContext completeTransition:YES];
-      }];
+  // Opacity animation
+  animatingView.alpha = startingAlpha;
+  [UIView animateWithDuration:self.opacityAnimationDuration
+                        delay:0
+                      options:options
+                   animations:^{
+                     animatingView.alpha = endingAlpha;
```

Comment:
Is this the scrim opacity or the dialog opacity? Both?

Reply:
Dialog – will update `animatingView` to `dialogView` for clarity